src/main/java/ui/AvatarUi.java

```diff
@@ -3,46 +3,31 @@
-            "███▓███▓█▓▓▓▓▓▓▓▓▓▓█▓▓█▓▓▓▓▓▓█▓░░░░░░▓████▒░░░░░░██▓▓▓▓▓██▓▓█▓▓▓▓▓▓▓▓▓▓▓████▒▒▒▒\n" +
-            "▒▒▒▒███▓█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓██░░░░░░█████▓░░░░░▒██▓▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓▓█▓███▓▓▓▓\n" +
-            "▓▓▓▓████▓█▓▓▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓██▓░░░░░██████░░░░░▓██▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓▓▓██████▓██\n" +
-            "████████▓██▓▓▓▓▓▓▓▓▓██▓▓▓▓▓▓▓██▓▒░░░▒██████▒░░░▒▓██▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓██▓█████▓▓▓\n" +
-            "▓▓▓████████████▓▓▓▓▓▓██▓▓▓▓▓▓██▓▓░░░▓██████▒░░░▓▓██▓▓▓▓▓▓███▓▓▓▓▓▓▓███████████▓▓\n" +
-            "░░░█████████████████▓██████████▓▓▓░░▓██████▓░░▓▓▓███████████████████████████████\n" +
-            "▒▒▓█████████████████████████████▓▓▒░▓███████░▒█▓▓█████████████████████████████▓▓\n" +
-            "█▓█████████████████████████████████▒▓███████░▓▓▓████████████▓▓▓▓▓█████████████▒▒\n" +
-            "█▒██████████████████████████████████████████▓████████████▓▓▓▓█████████████████▓▒\n" +
-            "█▒████████████████████████████████████████████████████████████████████████████▓▒\n" +
-            "▓▓█████████████████████████████████████████████████████████████████████████████▒\n" +
-            "▓▓█████████████████████████████████████████████████████████████████████████████▒\n";
+            "▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒\n" + 
+            "░░░░░░░░░░░░░░░░▒▒▓███▓▓▓▓▓▓▓█▓▓▒░░░░░░░░░░░░░░░░░\n" + 
+            "░░░░░░░░░░░▒▒▒▒▒▓███▓▓▓▓▓▓███▓▓▓█▒▒▒▒▒▒▒░░░░░░░░░░\n" + 
+            "▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒█████▓▓▓█▓▓▓▓▓▓██▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒\n" + 
+            "░░░░░░░░░░░░░░░░███▓▒▒▒▒▒▒▒▒▒▒▓██▒░░░░░░░░░░░▒░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░███▓▒▒▒▒▒▒▒▒▒▒▓██░░░░░░░░░░░░░░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░▒█▓▓███▓▒▒▓██▓▓██░░░░░░░░░░░░░░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░▓█▓▒▒▒▒▒▒▒▒▒▒▒▒█▓░░░░░░░░░░░░░░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░░▓▓▓▒▒▒▒▒▒▒▒▒▒▓▓▒░░░░░░░░░░░░▒░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░░░▒▓▒▒▒▒▓▓▒▒▒▓▓░░░░░░░░░░░░░░▒░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░░░░▓▓▒▒▓▓▓▓▒▓▓▒░░░░░░░░░░░░░░▒░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░░░░▓▓▓▒▒▒▒▒▒▓▓░░░░░░░░░░░░░░░▒░░▒░\n" + 
+            "░░░░░░░░░░░░░░░░░░▒▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░▒░░▒░\n" + 
+            "░▒░░░░░▒░░░░░░░░▒▓█▒▒▒▓▓▓▓▓▒▒▒▒█▓▒░░░░░░░░░░░▒░░▒░\n" + 
+            "░▒░░▒▓▓▓▒░░░▒▒▓███▓▒░░░▒▓▓░░░▒▒████▓▒▒░░░░░░░▒░░▒░\n" + 
+            "▓▓▒░▒▓▓▓▓▓▓██████▓▓▒░░▒████░░░▒█▓██████▓▓▓▓▓▒▒▒▒▒▒\n" + 
+            "▓▓▓▓██████▓▓▓██▓▓▓▓░░░░▓██░░░░▒█▓▓▓█▓▓▓▓██████▓▓▓▒\n" + 
+            "▓▓████▓▓▓▓▓▓█▓▓▓▓▓█░░░░▓██▒░░░▒█▓▓▓▓▓▓▓▓▓▓▓▓███▓▓▓\n" + 
+            "▒▒▓███▓▓▓▓▓▓▓▓▓▓▓▓█▒░░░███▓░░░▓█▓▓▓▓▓▓▓▓▓▓▓▓▓███▓▓\n" + 
+            "███████▓▓▓▓▓██▓▓▓▓█▓░░░████░░▒██▓▓▓▓█▓▓▓▓▓▓█████▓▓\n" + 
+            "▒▒████████▓▓▓██▓▓▓█▓▓░▒████░░▓████▓██▓▓▓██████████\n" + 
+            "▒▓██████████████████▓▒▒████▒▓▓█████████▓████████▓▓\n" + 
+            "▓▓████████████████████▓████▓████████▓▓▓█████████▓▒\n" + 
+            "▓████████████████████████████████████████████████▒\n" + 
+            "▓████████████████████████████████████████████████▒\n";
```

Comment:
Looks good! We may want to test how it appears on different computers after v2.0.